src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
Let's look at source of Google Blink core (one of the highest quality open source code): https://chromium.googlesource.com/chromium/blink/+/master/Source/core/ 
I looked up a few files and almost everywhere for non-fundamental types used the prefix form. There is Google C++ Style Guide: https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Preincrement_and_Predecrement
We should not always rely on the optimizer. The experience of re-engineering of code can say that even high-quality optimizer is often unable to cope with their work and do not generate optimal code.